framework/integration/test/basic/basic_test.go

```diff
@@ -15,15 +17,17 @@ import (
 // operations. The test verifies that finalizers are added and removed as
 // expected. It does not cover correct behavior with reconciliation.
 func Test_Finalizer_Integration_Basic(t *testing.T) {
-	mustSetup()
-	defer mustTeardown()
-
-	operatorName := "test-operator"
 	configMapName := "test-cm"
 	expectedFinalizers := []string{
 		"operatorkit.giantswarm.io/test-operator",
 	}
-	operatorkitFramework, err := newFramework(operatorName)
+	operatorName := "test-operator"
+	testNamespace := "finalizer-integration-basic-test"
+
+	integration.MustSetup(testNamespace)
```

Comment:
I am not using a `TestMain` to do this stuff currently, can someone explain to me why we use `TestMain` if it's just going to be one testcase per Main anyway?